Packs/Gurucul/Integrations/GuruculGRA/GuruculGRA.yml

```diff
@@ -773,6 +781,9 @@ script:
     - contextPath: Gra.Cases.status
       description: satus
       type: string
+    - contextPath: Gra.Cases.anomalies
```

Comment:
since i don't see a relevant change in the code, i suppose this was always outputted by the command and was just missing from the YML ?

Reply:
This field is used only for display multiple anomalies on the screen.

Reply:
not sure i understand what you mean by "display multiple anomalies on the screen", can you elaborate please?

Reply:
In Gra, cases having multiple anomalies are stored in the anomalies column.
Cases fetched from GRA and incidents created in demisto then anomalies related behavior asked in configuration 
Case 1.Incident create with the case with single  anomaly(as per anomaly count, incidents will create in demisto) 
here we created multiple columns like anomaly name, status, etc OR
Case 2.Incident create with the case with multiple anomalies(Per case only one incident create in demisto)
Here we used the anomalies column to store multiple anomalies  
At a time only one case is executed when we fetch case data from GRA
Hope I clarified, Still need more information then reply or mail me so I will clear your doubt